src/analysis/calculus/fderiv.lean

```diff
@@ -1381,6 +1398,16 @@ begin
   exact has_fderiv_at.comp x hg.has_fderiv_at hf.has_fderiv_at
 end
 
+lemma fderiv.comp_fderiv_within {g : F → G}
+  (hg : differentiable_at 𝕜 g (f x)) (hf : differentiable_within_at 𝕜 f s x)
+  (hxs : unique_diff_within_at 𝕜 s x) :
+  fderiv_within 𝕜 (g ∘ f) s x =
+    continuous_linear_map.comp (fderiv 𝕜 g (f x)) (fderiv_within 𝕜 f s x) :=
```

Comment:
Does `(fderiv 𝕜 g (f x)).comp` work here?

Reply:
Yes. Changed this lemma and the nearby ones.